Example/android/app/build.gradle

```diff
@@ -240,6 +240,8 @@ dependencies {
         exclude group:'com.facebook.flipper'
     }
 
+    compile project(":react-native-android-keyboard-adjust")
```

Comment:
hm, why? it doesn't support auto-linking?

Reply:
@savelichalex It seems so.

Reply:
Same for Surf, they do not support it indeed.

Reply:
btw I believe it should `implementation` instead of `compile`, AFAIR it should complain about it in AS and during a build